kasa/smart/modules/temperaturesensor.py

```diff
@@ -29,6 +29,18 @@ def _initialize_features(self) -> None:
                 type=Feature.Type.Sensor,
             )
         )
+        self._add_feature(
```

Comment:
Was this just moved (to clean up the ordering)?

Reply:
Reverted ordering change